charts/consul/values.yaml

```diff
@@ -572,7 +572,7 @@ global:
   # The name (and tag) of the consul-dataplane Docker image used for the
   # connect-injected sidecar proxies and mesh, terminating, and ingress gateways.
   # @default: hashicorp/consul-dataplane:<latest supported version>
-  imageConsulDataplane: hashicorp/consul-dataplane:1.0.0-beta2
+  imageConsulDataplane: duptonhashicorp/consul-dataplane:distroless-f7c2a05
```

Comment:
We need a "real" image 😅 (this one is from my laptop and has some extra debugging nonsense in)